connect/runtime/src/test/java/org/apache/kafka/connect/util/clusters/WorkerHandle.java

```diff
@@ -54,24 +55,6 @@ public void stop() {
         worker.stop();
     }
 
-    /**
-     * Determine if this worker is running.
-     *
-     * @return true if the worker is running, or false otherwise
-     */
-    public boolean isRunning() {
-        return worker.isRunning();
-    }
-
-    /**
-     * Get the workers's name corresponding to this handle.
-     *
-     * @return the worker's name
-     */
-    public String name() {
```

Comment:
This appears to have been unused. I can't imagine a situation where you'd want to programmatically search for the name, so it should probably only be visible in the toString() :+1: